### ISSUE TITLE
Unable to create a volume for solr data

### ISSUE THREAD

**Issue by grozeille** [HariSekhon/Dockerfiles#17]
I tried to create volumes on /solr/example/cloud/ but the permissions are incorrect.
Could you please add a "chmod" in the DockerFile to set the right permissions to  /solr/example/cloud/  before starting solr ?

**Comment by HariSekhon** [comment]
Hi @grozeille 

Thanks for raising this ticket.

I changed to chown solr /solr* only to realise this was already done - are you running as root or solr or some other user. If running as root entrypoint.sh will drop to solr user so it should just work. There are tests running against this repo which run solr -e cloud (Advanced Nagios Plugins Collections runs a series of API tests against the container after it's up).

What is the exact error you get and what user are you running as? Which tag version are you running, latest?

Hari

**Comment by HariSekhon** [comment]
Any update on this?
